.kokoro

```diff
@@ -197,7 +197,18 @@ run_bazel_affected() {
       pushd github/repo >> /dev/null
     fi
 
-    TARGET="$(scripts/affected_targets $rule_kind)"
+    if [ -n "$KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH" ]; then
+      target_branch="$KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH"
+    else
+      target_branch=develop
+    fi
+    if [ $(git branch --list "$target_branch") ]; then
+      base_sha=$(git merge-base "$target_branch" HEAD)
+    else
+      base_sha="HEAD^"
+    fi
+
+    TARGET="$(scripts/affected_targets $rule_kind $base_sha...HEAD)"
```

Comment:
Is "rule_kind" the same idea as "type" in the other script? I'm trying to figure out what the valid values for "type" are below, but somehow missed the documentation.

Reply:
Fixed the naming so they're consistent across both files.